tests/fs.md

```diff
@@ -18,7 +18,6 @@ open Eio.Std
 let ( / ) = Path.( / )
 
 let run ?clear:(paths = []) fn =
-  Eio_main.run @@ fun env ->
```

Comment:
Why did you remove this @webbunivAdmin ? It is very important as it is the effect handler for Eio code. Without this none of this tests will work.